src/client/client.js

```diff
@@ -17,35 +17,31 @@ const initialState = iMap([
   ['form', form]
 ]);
 
+// const authToken = localStorage.getItem(authTokenName);
+
+if (authToken) {
+
+}
 // Create the store:
 const store = makeStore(initialState);
 
 // Create the Cashay singleton:
 const cashaySchema = require('cashay!../server/utils/getCashaySchema.js');
 const authToken = localStorage.getItem(localStorageVars.authTokenName);
-const cashayHttpTransport = new HTTPTransport(
-  getGraphQLUri(),
-  {
-    headers: {
-      'Content-Type': 'application/json',
-      Authorization: `Bearer ${authToken}`
-    }
-  }
-);
-const cashayParams = {
+
+const cashayHttpTransport = new ActionHTTPTransport(authToken);
+
+cashay.create({
   store,
   schema: cashaySchema,
   getToState: reduxStore => reduxStore.getState().get('cashay'),
   transport: cashayHttpTransport
-};
-
-// export the Cashay singleton:
-export const cashay = new Cashay(cashayParams);
+});
 
 
 render(
   <AppContainer>
-    <Root store={store} />
+    <Root store={store}/>
```

Comment:
Style note: I like the space before /. For me, makes it clearer to see. You opposed?